src/nock/matcher.utils.ts

```diff
@@ -72,7 +72,7 @@ function composeBuffer(response: any[]): Buffer {
 
 export async function decodeResponse(
   response: unknown,
-  contentEncoding: string
+  contentEncoding = 'gzip'
```

Comment:
Is this necessary? I don't think it's safe to assume encoding by default.

Reply:
No, you're right, I'll remove it.